src/Oauth2CredentialManagers/FileStore.php

```diff
@@ -131,4 +131,13 @@ protected function data($key = null)
 
         return empty($key) ? $cacheData : ($cacheData[$key] ?? null);
     }
+
+    public function delete(): void
+    {
+        $ret = $this->files->delete($this->filePath);
```

Comment:
Can you please check if the file exists first, if it doesn't then just do an early return.

To ensure both the filestore and the cachestore act the same we should only throw an error if we fail to delete the file.
The file missing in the first place isn't really an error, just a useless operation.

Similarly, with the cache adapter if you call delete on a key that doesn't exist it will just do nothing.